distribution/lib/Standard/Base/0.0.0-dev/src/Data/Time/Time_Period.enso

```diff
@@ -1,17 +1,19 @@
-from Standard.Base import all
+import project.Data.Time.Time_Of_Day.Time_Of_Day
+import project.Data.Time.Date_Time.Date_Time
+from project.Data.Boolean import Boolean, True, False
 
 polyglot java import org.enso.base.Time_Utils
 polyglot java import java.time.temporal.ChronoUnit
-
-# TODO Dubious constructor export
-from project.Data.Time.Time_Period.Time_Period import all
-from project.Data.Time.Time_Period.Time_Period export all
+polyglot java import java.time.temporal.TemporalUnit
 
 ## Represents a period of time of a day or shorter.
 type Time_Period
```

Comment:
I feel like with the new changes where we have `Duration+Period`, this name and comment may be confusing.

Maybe we should rephrase to
```suggestion
## Represents a unit of time of a day or shorter.
type Time_Period
```